tiledb/api/c_api/group/group_api_external.h

```diff
@@ -630,12 +632,28 @@ TILEDB_EXPORT capi_return_t tiledb_group_get_query_type(
  * @param recursive should we recurse into sub-groups
  * @return  `TILEDB_OK` for success and `TILEDB_ERR` for error.
  */
-TILEDB_EXPORT capi_return_t tiledb_group_dump_str(
+TILEDB_DEPRECATED_EXPORT capi_return_t tiledb_group_dump_str(
     tiledb_ctx_t* ctx,
     tiledb_group_t* group,
     char** dump_ascii,
     const uint8_t recursive) TILEDB_NOEXCEPT;
 
+/**
+ * Dump a string representation of a group.
+ *
```

Comment:
It's more clear to add an `@note The caller must free the returned char (dump_ascii).` here, before the params, rather than an inline note that could easily be overlooked.

Reply:
Updated documentation to match other dump APIs. https://github.com/TileDB-Inc/TileDB/blob/abca93c6340ee936d435b3439865b5bc7e1ec11b/tiledb/api/c_api/dimension/dimension_api_external.h#L298-L317